lib/Mount/CacheRootPermissionsMask.php

```diff
@@ -15,14 +15,14 @@
 class CacheRootPermissionsMask extends CacheWrapper {
 	public function __construct(
 		ICache $cache,
-		private int $mask,
+		private readonly int $mask,
 	) {
 		parent::__construct($cache);
 	}
 
 	protected function formatCacheEntry($entry): ICacheEntry|false {
 		$path = $entry['path'];
```

Comment:
```suggestion
		$path = (string)$entry['path'];
```
It’s better to cast early rather than when the variable is used.
It’s the same for most of the other cast changes actually.